lms/djangoapps/courseware/access.py

```diff
@@ -323,6 +324,14 @@ def can_load():
         if descriptor.visible_to_staff_only and not _has_staff_access_to_descriptor(user, descriptor, course_key):
             return False
 
+        # if course has pre-requisite course not passed by user
+        if settings.FEATURES['ENABLE_PREREQUISITE_COURSES'] \
+                and not _has_staff_access_to_descriptor(user, descriptor, course_key) \
+                and hasattr(descriptor, 'pre_requisite_courses') and descriptor.pre_requisite_courses \
+                and not user.is_anonymous() \
+                and get_pre_requisite_courses_not_completed(user, [(descriptor, None)]):
```

Comment:
passing a list of course_keys user is enrolled in to `get_pre_requisite_courses_not_completed` instead of list of tuples.